libclamav_rust/Cargo.toml

```diff
@@ -26,6 +26,13 @@ inflate = "0.4"
 bzip2-rs = "0.1"
 byteorder = "1.5"
 delharc = "0.6"
+clam-sigutil = { path = "../../clamav-signature-util" }
```

Comment:
This is temporarily pointing a copy of the rust clam-sigutil project in my workspace. Will change it to a github URL soon.